src/libmongoc/src/mongoc/mongoc-client.h

```diff
@@ -268,6 +268,9 @@ mongoc_client_enable_auto_encryption (mongoc_client_t *client,
                                       mongoc_auto_encryption_opts_t *opts,
                                       bson_error_t *error);
 
+MONGOC_EXPORT (int64_t)
+mongoc_client_get_timeout_ms (const mongoc_client_t *client);
+
```

Comment:
Why not add `mongoc_client_set_timeout_ms`? There's plenty of setters for key-values that can also be set in the URI. For example, `appname`, `readPrefs`, etc.

Reply:
I originally did have a `mongoc_client_set_timeout_ms`, but I removed it.  The feedback I got from @jmikola and @kevinAlbs was that we should only allow timeoutMS to be set on clients through the URI, at least for now, since that helps avoid some of the weirdness of keeping properties of pooled clients consistent with their parent pools.  The spec does not mandate that there be a setter for clients, just that the timeout "MUST be configurable at the level of a MongoClient."

Reply:
Oh, [CDRIVER-3537](https://jira.mongodb.org/browse/CDRIVER-3537) right? Gotcha.